src/In.ProjectEKA.HipService/DataFlow/Database/Migrations/20200226055335_AddHealthInformationTable.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace In.ProjectEKA.HipService.DataFlow.Database.Migrations
+{
+    using System;
+    using Microsoft.EntityFrameworkCore.Migrations;
+
+    public partial class AddHealthInformationTable : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5067092)